data/tmpl/utility/content3.html

```diff
@@ -18,7 +18,7 @@
           <button class="button" id="template2">Try Shiretoko Fox</button>
         </div>
         <button class="button reset" id="default">Reset to default</button>
-        <a href="https://addons.mozilla.org/firefox/themes/?utm_source=fxonboarding&amp;utm_medium=firefox-browser&amp;utm_campaign=onboardingv1" target="_blank">Explore other themes</a>
+        <a id="exploreThemes" href="">Explore other themes</a>
```

Comment:
@FrancescoSTL we have been still placing the same URL in href in other sidebars so that when you mouse over the link, it shows it in Firefox.